GCMFormTableView/GCMFormTableView/GCMFormTableView.h

```diff
@@ -0,0 +1,19 @@
+//
+//  GCMFormTableView.h
+//  GCMFormTableView
+//
+//  Created by Eduardo Arenas on 10/16/15.
+//  Copyright © 2015 GameChanger. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for GCMFormTableView.
+FOUNDATION_EXPORT double GCMFormTableViewVersionNumber;
+
+//! Project version string for GCMFormTableView.
+FOUNDATION_EXPORT const unsigned char GCMFormTableViewVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <GCMFormTableView/PublicHeader.h>
```

Comment:
It's possible.... I think if you use Cocoapods it doesn't matter. According to what I read, you have to declare the public headers in the Podspec for them to be available, and if you don't it will make every header public, which seems like what we want.